src/shared/components/getting-started-section/index.module.css

```diff
@@ -66,6 +66,104 @@
     }
 }
 
+.LiveSnippet {
+    width: 60%;
+    overflow: hidden;
+    margin: auto;
+    margin-bottom: 75px;
+    border-radius: 2px;
+}
+
+.LiveSnippet pre[class*="language-"] {
+    margin: 0;
+    border-radius: 0;
+}
+
+.LiveSnippet-title {
```

Comment:
Classes should all be camelCase.. as a bonus they are easier to use in the JS side.